packages/kotti-table/src/Table.vue

```diff
@@ -46,9 +46,10 @@ export default {
 	components: { TableBody, TableHeader, TableColumn },
 	name: 'KtTable',
 	props: {
+		id: { default: null, type: String },
 		rowKey: { type: String },
-		rows: { required: true, type: Array },
-		columns: { default: () => [], type: Array },
+		rows: { default: () => [], type: Array },
```

Comment:
Does a table without `rows` make sense?

Reply:
no, but I recall needing a row array to always exist for the table internal state, so it was either initialise it here, or initialise it in the store.